src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
```

Comment:
All `isMutating()` methods should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves.